scss/_stepped-process.scss

```diff
@@ -2,123 +2,141 @@
+  --#{$prefix}stepped-process-link-color: #{$step-link-color};
+  --#{$prefix}stepped-process-link-active-color: #{$step-link-active-color};
+  --#{$prefix}stepped-process-link-marker: counter(var(--bs-stepped-process-counter)) "\A0";
+  --#{$prefix}stepped-process-link-text-decoration: #{$step-link-text-decoration};
+  // scss-docs-end stepped-process-css-vars
+
+  @include media-breakpoint-up(sm) {
+    --#{$prefix}stepped-process-item-padding-x: 0;
+    --#{$prefix}stepped-process-item-margin-end: 0;
+    --#{$prefix}stepped-process-item-drop-shadow: #{$step-item-drop-shadow};
+    --#{$prefix}stepped-process-item-arrow-width: #{$step-item-arrow-width};
+  }
+
+  @include media-breakpoint-up(xl) {
+    --#{$prefix}stepped-process-link-width: none;
+    --#{$prefix}stepped-process-link-marker: counter(var(--bs-stepped-process-counter)) ".\A0";
+  }
+
   ol {
     display: flex;
     padding: 0;
     margin: 0;
     overflow: hidden;
-    font-size: $small-font-size;
-    font-weight: $font-weight-bold;
+    font-size: var(--#{$prefix}stepped-process-font-size);
+    font-weight: var(--#{$prefix}stepped-process-font-weight);
     text-align: center;
     list-style: none;
-    counter-reset: $stepped-process-counter;
+    counter-reset: var(--#{$prefix}stepped-process-counter);
   }
 }
 
 .stepped-process-item {
+  position: relative;
   display: flex;
   flex: 1;
-  padding: $step-item-padding;
-  margin-right: $step-item-margin-end;
-  counter-increment: $stepped-process-counter;
-  background-color: $step-item-bg;
+  padding: var(--#{$prefix}stepped-process-item-padding-y) var(--#{$prefix}stepped-process-item-padding-x);
+  counter-increment: var(--#{$prefix}stepped-process-counter);
+  background-color: var(--#{$prefix}stepped-process-item-bg);
+  filter: var(--#{$prefix}stepped-process-item-drop-shadow);
+
+  &:not(:last-child) {
+    margin-right: var(--#{$prefix}stepped-process-item-margin-end);
+
+    &::after {
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      left: subtract(100%, 1px);
+      z-index: -1;
+      width: var(--#{$prefix}stepped-process-item-arrow-width);
+      clip-path: var(--#{$prefix}stepped-process-item-arrow-shape);
+      content: "";
+      background-color: inherit;
+    }
+  }
 
   &.active {
-    background-color: $step-item-active-bg;
+    background-color: var(--#{$prefix}stepped-process-item-active-bg);
   }
 
   .active ~ & {
-    background-color: $step-item-next-bg;
+    background-color: var(--#{$prefix}stepped-process-item-next-bg);
+  }
+
+  & + & {
+    padding-left: var(--#{$prefix}stepped-process-item-arrow-width);
   }
 }
 
 .stepped-process-link {
-  flex: 1 0 $step-link-width;
-  max-width: $step-link-width;
+  flex: 1 0 var(--#{$prefix}stepped-process-link-width);
+  max-width: var(--#{$prefix}stepped-process-link-width);
   margin: auto;
   overflow: hidden;
-  line-height: 1;
-  color: $step-link-color;
+  line-height: 1; // not enough
```

Comment:
Needs to be tackled in another PR ?

Reply:
Yes could you please create an issue about that and remove this comment?

Reply:
#1337